### PR TITLE
mongosync script

### PR DESCRIPTION
**Pull request by yCobanoglu**

### CODE REVIEW

**Comment by yCobanoglu** [comment]
@jlaasonen everything is working **besides** environment variables. They aren't passed to the container/ the script. If i hardcode the env variables in the mongosync.sh  it works but if i try to access then through docker-compose it fails.

File: mongosync/mongosync.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+printenv
```
Comment (jlaasonen) [inline mongosync/mongosync.sh:3]: The environment contains secrets. Do not print it.

**Review by jlaasonen — CHANGES_REQUESTED** [review]
You removed the old script, but the new one does not have all the feature. E.g. allow any DB or use different host and user for source and target.

Either implement the missing features and update the README or move the script to own repository.

**Comment by jlaasonen** [comment]
[Synching Ebl-dev database with bash script](https://trello.com/c/eUptlA9M/470-synching-ebl-dev-database-with-bash-script)

**Comment by jlaasonen** [comment]
Moved to own repository: https://github.com/ElectronicBabylonianLiterature/mongosync

**Pull request closed without merging**